src/rgw/rgw_ldap.h

```diff
@@ -83,10 +83,8 @@ namespace rgw {
 			      (void*) &ldap_ver);
 	if (ret == LDAP_SUCCESS) {
 	  ret = ldap_simple_bind_s(tldap, dn, pwd.c_str());
-	  if (ret == LDAP_SUCCESS) {
-	    (void) ldap_unbind(tldap);
-	  }
 	}
+	(void) ldap_unbind(tldap);
```

Comment:
yes, it looks like, ldap_unbind() can be called so long as any of ldap_initialize(), ldap_open, ... have been called on the handle, and frees the handle

Reply:
If this looks good to you, can this be merged and backported soon?
This issue has quite the impact on our central infrastructure unfortunately.

Reply:
Could you please elaborate more on what you mean by "sanity testing"?